"","rowname","error"
"1","ErrorMessage","invalid 'envir' argument"
"2","ErrorType","error"
"3","ErrorMessage6","unused argument (alist(x = ))"
"4","ErrorMessage7","argument ""names"" is missing, with no default"
"5","ErrorMessage8","the condition has length > 1"
"6","ErrorMessage10","invalid 'allow_' value"
"7","ErrorMessage18","'names' must be a character vector"
"8","ErrorMessage23","argument ""sym"" is missing, with no default"
"9","ErrorMessage24","argument ""fun"" is missing, with no default"
"10","ErrorMessage25","argument ""env"" is missing, with no default"
"11","ErrorMessage26","not a symbol"
"12","ErrorMessage30","not a function"
"13","ErrorMessage31","'...' used in an incorrect context"
"14","ErrorMessage32","argument ""f"" is missing, with no default"
"15","ErrorMessage33","object 'x' of mode 'function' was not found"
"16","ErrorMessage35","argument ""FUN"" is missing, with no default"
"17","ErrorMessage44","argument 'MoreArgs' of 'mapply' is not a list or pairlist"
"18","ErrorMessage57","unused argument (alist(FUN = ))"
"19","ErrorMessage58","argument ""x"" is missing, with no default"
"20","ErrorMessage59","only 0's may be mixed with negative subscripts"
"21","ErrorMessage60","invalid subscript type 'list'"
"22","ErrorMessage61","'X' must have named dimnames"
"23","ErrorMessage62","'x' is not an array"
"24","ErrorMessage64","unused arguments (alist(FUN = , quietly = ))"
"25","ErrorMessage71","unused arguments (alist(nrow = , ncol = ))"
"26","ErrorMessage72","argument ""nc"" is missing, with no default"
"27","ErrorMessage76","invalid 'nrow' value (< 0)"
"28","ErrorMessage77","non-numeric matrix extent"
"29","ErrorMessage80","argument ""table"" is missing, with no default"
"30","ErrorMessage81","'list' object cannot be coerced to type 'double'"
"31","ErrorMessage82","NAs introduced by coercion"
"32","ErrorType82","warning"
"33","ErrorMessage88","argument ""arg"" is missing, with no default"
"34","ErrorMessage89","'arg' must be NULL or a character vector"
"35","ErrorMessage97","invalid argument type"
"36","ErrorMessage98","unused argument (alist(substitute = ))"
"37","ErrorMessage99","invalid 'definition' argument"
"38","ErrorMessage100","invalid 'call' argument"
"39","ErrorMessage115","argument is not interpretable as logical"
"40","ErrorMessage119","object '.Generic' not found"
"41","ErrorMessage121","non-numeric-alike variable(s) in data frame: 1"
"42","ErrorMessage144","invalid 'byrow' argument"
"43","ErrorMessage153","argument 1 is empty"
"44","ErrorMessage154","no non-missing arguments to max; returning -Inf"
"45","ErrorMessage159","'arg' should be one of “random”, “first”, “last”"
"46","ErrorMessage164","argument is not numeric or logical: returning NA"
"47","ErrorMessage171","'trim' must be numeric of length one"
"48","ErrorMessage186","'origin' must be supplied"
"49","ErrorMessage187","do not know how to convert 'x' to class “POSIXct”"
"50","ErrorMessage188","character string is not in a standard unambiguous format"
"51","ErrorMessage191","argument ""from"" is missing, with no default"
"52","ErrorMessage192","'from' must be raw or character"
"53","ErrorMessage197","'arg' should be one of “gzip”, “bzip2”, “xz”, “none”"
"54","ErrorMessage206","'arg' should be one of “unknown”, “gzip”, “bzip2”, “xz”, “none”"
"55","ErrorMessage209","unused argument (x)"
"56","ErrorMessage210","unused argument (df)"
"57","ErrorMessage211","unused argument (""df"")"
"58","ErrorMessage214","argument ""y"" is missing, with no default"
"59","ErrorMessage218","'by' must match numbers of columns"
"60","ErrorMessage236","argument ""..1"" is missing, with no default"
"61","ErrorMessage237","
"
"62","ErrorType237","message"
"63","ErrorMessage244","invalid first argument"
"64","ErrorMessage260","second argument must be an environment"
"65","ErrorMessage263","no non-missing arguments to min; returning Inf"
"66","ErrorMessage265","'missing' can only be used for arguments"
"67","ErrorMessage267","non-numeric argument to function"
"68","ErrorMessage269","argument 2 is empty"
"69","ErrorMessage270","2 arguments passed to 'Mod' which requires 1"
"70","ErrorMessage278","argument ""value"" is missing, with no default"
"71","ErrorMessage284","object 'as.df' of mode 'function' was not found"
"72","ErrorMessage287","no applicable method for 'months' applied to an object of class ""name"""
"73","ErrorMessage288","no applicable method for 'months' applied to an object of class ""data.frame"""
"74","ErrorMessage289","no applicable method for 'months' applied to an object of class ""character"""
"75","ErrorMessage291","invalid 'trim' argument"
"76","ErrorMessage293","'list' object cannot be coerced to type 'logical'"
"77","ErrorMessage311","$ operator is invalid for atomic vectors"
"78","ErrorMessage314","1 argument passed to 'names<-' which requires 2"
"79","ErrorMessage318","unused arguments (alist(xx = , tz = , cl = ))"
"80","ErrorMessage319","unused arguments (alist(xx = x, tz = , cl = ))"
"81","ErrorMessage320","unused arguments (alist(xx = x, tz = x, cl = ))"
"82","ErrorMessage321","unused arguments (alist(xx = x, tz = df, cl = ))"
"83","ErrorMessage322","unused arguments (alist(xx = x, tz = ""df"", cl = ))"
"84","ErrorMessage323","unused arguments (alist(xx = df, tz = ""df"", cl = ))"
"85","ErrorMessage324","unused arguments (alist(xx = ""df"", tz = ""df"", cl = ))"
"86","ErrorMessage325","unused arguments (xx = ""df"", tz = ""df"", cl = x)"
"87","ErrorMessage326","unused arguments (xx = ""df"", tz = ""df"", cl = df)"
"88","ErrorMessage327","unused arguments (xx = ""df"", tz = ""df"", cl = ""df"")"
"89","ErrorMessage332","attempt to set an attribute on NULL"
"90","ErrorMessage334","unused argument (alist(exports = ))"
"91","ErrorMessage335","argument ""ns"" is missing, with no default"
"92","ErrorMessage336","not an environment"
"93","ErrorMessage342","unused argument (alist(self = ))"
"94","ErrorMessage343","argument is missing, with no default"
"95","ErrorMessage344","unused argument (alist(classes = ))"
"96","ErrorMessage345","argument ""vars"" is missing, with no default"
"97","ErrorMessage348","'The name of the object (e.g,. a class or generic function) to find in the meta-data' must be a single string (got an object of class ""numeric"")"
"98","ErrorMessage352","'The name of the object (e.g,. a class or generic function) to find in the meta-data' must be a single string (got an object of class ""integer"")"
"99","ErrorMessage353","not a namespace"
"100","ErrorMessage355","there is no package called ‘df’"
"101","ErrorMessage358","unused argument (alist(imports = ))"
"102","ErrorMessage359","argument ""self"" is missing, with no default"
"103","ErrorMessage371","there is no package called ‘x’"
"104","ErrorMessage372","unused argument (alist(methods = ))"
"105","ErrorMessage375","invalid 'pos' argument"
"106","ErrorMessage383","no item called ""df"" on the search list"
"107","ErrorMessage386","1 argument passed to 'nargs' which requires 0"
"108","ErrorMessage391","invalid 'type' argument"
"109","ErrorMessage403","'enclos' must be an environment"
"110","ErrorMessage411","no loop for break/next, jumping to top level"
"111","ErrorMessage414","'function' is not a function, but of type 8"
"112","ErrorMessage421","unused arguments (alist(msgid = , msgid_plural = ))"
"113","ErrorMessage422","argument ""n"" is missing, with no default"
"114","ErrorMessage423","argument ""msg1"" is missing, with no default"
"115","ErrorMessage424","argument ""msg2"" is missing, with no default"
"116","ErrorMessage425","invalid 'n' argument"
"117","ErrorMessage437","argument ""obj"" is missing, with no default"
"118","ErrorMessage445","'A' must be a numeric matrix"
"119","ErrorMessage451","argument ""path"" is missing, with no default"
"120","ErrorMessage452","invalid 'path' argument"
"121","ErrorMessage462","invalid 'length' argument"
"122","ErrorMessage467","invalid version specification ‘-1’, ‘-Inf’, ‘NaN’, ‘Inf’"
"123","ErrorMessage468","'length(x) = 6 > 1' in coercion to 'logical(1)'"
"124","ErrorMessage469","invalid 'y' type in 'x && y'"
"125","ErrorMessage481","NAs introduced by coercion to integer range"
"126","ErrorMessage498","0 arguments passed to 'oldClass<-' which requires 2"
"127","ErrorMessage499","1 argument passed to 'oldClass<-' which requires 2"
"128","ErrorMessage502","invalid filename argument"
"129","ErrorMessage504","‘df’ is not a directory"
"130","ErrorMessage505","invalid 'lifo' argument"
"131","ErrorMessage507","invalid 'add' argument"
"132","ErrorMessage511","unused argument (alist(warn = ))"
"133","ErrorMessage512","no applicable method for 'open' applied to an object of class ""name"""
"134","ErrorMessage514","no applicable method for 'open' applied to an object of class ""c('double', 'numeric')"""
"135","ErrorMessage515","no applicable method for 'open' applied to an object of class ""data.frame"""
"136","ErrorMessage516","no applicable method for 'open' applied to an object of class ""character"""
"137","ErrorMessage517","argument ""con"" is missing, with no default"
"138","ErrorMessage519","'con' is not a connection"
"139","ErrorMessage543","no applicable method for 'open' applied to an object of class ""NULL"""
"140","ErrorMessage551","invalid 'text' argument"
"141","ErrorMessage553","object '.Method' not found"
"142","ErrorMessage619","'arg' should be one of “auto”, “shell”, “radix”"
"143","ErrorMessage622","cannot xtfrm data frames"
"144","ErrorMessage624","argument ""X"" is missing, with no default"
"145","ErrorMessage625","argument ""Y"" is missing, with no default"
"146","ErrorMessage627","argument ""df2"" is missing, with no default"
"147","ErrorMessage635","invalid version specification ‘-1’, ‘0’, ‘1’, ‘-Inf’, ‘NaN’, ‘Inf’"
"148","ErrorMessage641","unused argument (alist(lib.loc = ))"
"149","ErrorMessage642","argument ""pkgname"" is missing, with no default"
"150","ErrorMessage643","non-character argument"
"151","ErrorMessage646","'arg' should be one of “onLoad”, “attach”, “detach”, “onUnload”"
"152","ErrorMessage649","unused arguments (alist(pkg = , lib.loc = ))"
"153","ErrorMessage650","argument ""package.lib"" is missing, with no default"
"154","ErrorMessage652","unused argument (alist(pkg = ))"
"155","ErrorMessage653","argument ""package"" is missing, with no default"
"156","ErrorMessage654","argument ""lib.loc"" is missing, with no default"
"157","ErrorMessage665","argument 'x' must be raw, integer or logical"
"158","ErrorMessage668","'arg' should be one of “raw”, “integer”, “double”"
"159","ErrorMessage674","unused argument (alist(envir = ))"
"160","ErrorMessage676","argument is not an environment"
"161","ErrorMessage679","unused argument (envir = x)"
"162","ErrorMessage680","unused argument (envir = df)"
"163","ErrorMessage681","unused argument (envir = ""df"")"
"164","ErrorMessage683","unused arguments (alist(envir = , value = ))"
"165","ErrorMessage693","invalid 'n' value"
"166","ErrorMessage696","'file' must be a character string or connection"
"167","ErrorMessage698","invalid 'encoding' value"
"168","ErrorMessage700","argument 'encoding = ""df""' will be ignored"
"169","ErrorMessage712","unused arguments (alist(file = , env = ))"
"170","ErrorMessage719","package ‘df’ has no 'NAMESPACE' file"
"171","ErrorMessage724","invalid separator"
"172","ErrorMessage730","invalid 'collapse' argument"
"173","ErrorMessage738","unused arguments (alist(verbose = , lib.loc = ))"
"174","ErrorMessage739","none of the packages are loaded"
"175","ErrorMessage748","unused arguments (alist(command = , text = ))"
"176","ErrorMessage749","argument ""description"" is missing, with no default"
"177","ErrorMessage750","invalid 'description' argument"
"178","ErrorMessage755","cannot open pipe() cmd 'df': Invalid argument"
"179","ErrorMessage756","invalid 'encoding' argument"
"180","ErrorMessage760","'x' and 'y' lengths differ"
"181","ErrorMessage777","no arguments"
"182","ErrorMessage784","invalid 'na.rm' value"
"183","ErrorMessage797","invalid polynomial coefficient"
"184","ErrorMessage798","argument ""z"" is missing, with no default"
"185","ErrorMessage799","unimplemented type 'list' in 'polyroot'
"
"186","ErrorMessage800","unimplemented type 'character' in 'polyroot'
"
"187","ErrorMessage804","'list' object cannot be coerced to type 'integer'"
"188","ErrorMessage847","invalid 'decimal.mark' argument"
"189","ErrorMessage864","object not specified"
"190","ErrorMessage868","argument 1 (type 'list') cannot be handled by 'cat'"
"191","ErrorMessage871","no applicable method for 'conditionMessage' applied to an object of class ""c('double', 'numeric')"""
"192","ErrorMessage872","no applicable method for 'conditionMessage' applied to an object of class ""data.frame"""
"193","ErrorMessage873","no applicable method for 'conditionMessage' applied to an object of class ""character"""
"194","ErrorMessage884","invalid 'max' / getOption(""max.print""): df"
"195","ErrorMessage888","invalid printing digits -1"
"196","ErrorMessage917","invalid printing digits -2147483648"
"197","ErrorMessage927","undefined columns selected"
"198","ErrorMessage929","subscript out of bounds"
"199","ErrorMessage939","factor levels must be ""character"""
"200","ErrorMessage940","invalid 'quote' value"
"201","ErrorMessage947","invalid 'quote' argument"
"202","ErrorMessage953","invalid 'useSource' argument"
"203","ErrorMessage961","can only write character objects"
"204","ErrorMessage971","invalid 'right' argument"
"205","ErrorMessage1025","invalid 'sep' specification"
"206","ErrorMessage1036","invalid 'digits' parameter, allowed 1...22"
"207","ErrorMessage1050","invalid 'na.print' specification"
"208","ErrorMessage1051","invalid row labels"
"209","ErrorMessage1054","1 argument passed to 'proc.time' which requires 0"
"210","ErrorMessage1067","'x' must have named dimnames"
"211","ErrorMessage1085","unused argument (alist(value = ))"
"212","ErrorMessage1087","'dimnames' applied to non-array"
"213","ErrorMessage1099","NaNs produced"
"214","ErrorMessage1100","deriv = 2147483647 > 100 (= n_max)"
"215","ErrorMessage1101","non-numeric argument to mathematical function"
"216","ErrorMessage1105","unused arguments (alist(x = , value = ))"
"217","ErrorMessage1106","argument ""data"" is missing, with no default"
"218","ErrorMessage1107","argument ""connection"" is missing, with no default"
"219","ErrorMessage1108","invalid 'data' argument"
"220","ErrorMessage1112","'arg' should be one of “bytes”, “UTF-8”"
"221","ErrorMessage1126","invalid connection"
"222","ErrorMessage1130","one of ""yes"", ""no"", ""ask"" or ""default"" expected."
"223","ErrorMessage1137","unrecognized value of 'save'"
"224","ErrorMessage1140","NA/NaN/Inf in foreign function call (arg 1)"
"225","ErrorMessage1142","argument ""qr"" is missing, with no default"
"226","ErrorMessage1143","first argument must be a QR decomposition"
"227","ErrorMessage1162","argument is not a QR decomposition"
"228","ErrorMessage1211","unused arguments (alist(qr = , y = ))"
"229","ErrorMessage1212","argument ""a"" is missing, with no default"
